### ISSUE TITLE
Error loading "Sawyer, Eugene, 1934-2008"

### ISSUE THREAD

**Issue by johnjung** [johnjung/bmrcportal#152]
To duplicate the error, go to [https://bmrc.lib.uchicago.edu/portal/browse/?b=people](https://bmrc.lib.uchicago.edu/portal/browse/?b=people). Click [Sawyer, Eugene, 1934-2008 (4)](https://bmrc.lib.uchicago.edu/portal/search/?f=https%3A//bmrc.lib.uchicago.edu/people/Sawyer%252C%2BEugene%252C%2B1934-2008). The system errors out- why does this happen?

**Comment by MomoMoses** [comment]
I have no idea. Other names with commas all work. Other names with dates work. I'm surprised the system only finds 4 results, but maybe just for that configuration with the date span...?

**Comment by MomoMoses** [comment]
This error does NOT occur in bmrcportal-test. This error DOES occur in [bmrcportal](https://bmrcportal.lib.uchicago.edu/)

**Comment by johnjung** [comment]
Thank you very much for double checking. I noticed that one of the documents that was being returned for this browse from MarkLogic is a "binary blob" (in this case a .tmp file) instead of an XML file- that was causing an error in the search function.

I made some an improvement to catch stray documents like this in the function that removes every finding aid from MarkLogic in https://github.com/uchicago-library/bmrc/commit/7b5e4dcf19bcebd240e07b9a16a8caa6cf48ac83. When I push our recent changes to production and reload the finding aids there I'll be able to test this to see if it eliminates this error.
